core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -900,8 +900,8 @@ object ConsumerGroupCommand extends Logging {
     val CommandConfigDoc = "Property file containing configs to be passed to Admin Client and Consumer."
     val ResetOffsetsDoc = "Reset offsets of consumer group. Supports one consumer group at the time, and instances should be inactive" + nl +
       "Has 3 execution options: (default) to plan which offsets to reset, --execute to execute the reset-offsets process, and --export to export the results to a CSV format." + nl +
-      "Has the following scenarios to choose: --to-datetime, --by-period, --to-earliest, --to-latest, --shift-by, --from-file, --to-current. One scenario must be choose" + nl +
-      "To define the scope use: --all-topics or --topic. . One scope must be choose, unless you use '--from-file' scenario"
+      "Has the following scenarios to choose: --to-datetime, --to-earliest, --to-latest, --shift-by, --from-file, --to-current, --by-duration, --to-offset. One scenario must be choosen" + nl +
+      "To define the scope use: --all-topics or --topic. One scope must be choosen, unless you use '--from-file' scenario"
```

Comment:
--by-period seems obsolete. Removed it from usage.
Added  --by-duration and --to-offset.
Corrected spelling